sklearn/datasets/base.py

```diff
@@ -258,6 +258,15 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+    ----------
+    
+        .. versionadded:: 0.18
```

Comment:
Sorry for replying late.

I intend on sending the next PR this weekend (adding return_X_y option to load_breast_cancer dataset)
Should I take care of this ?